pkg/controller/importconfig/cluster_info.go

```diff
@@ -224,22 +224,18 @@ func buildImportSecret(ctx context.Context, clientHolder *helpers.ClientHolder,
 			WithManagedCluster(managedCluster).
 			WithKlusterletConfig(klusterletConfig).
 			WithPriorityClassName(priorityClassName)
-		yamlcontent, err = config.Generate(ctx, clientHolder)
+		yamlcontent, crdsV1YAML, err = config.Generate(ctx, clientHolder)
```

Comment:
How many crds are left in the secret after the refactor? crd.yaml and crdv1.yaml? or only the crd.yaml?

Reply:
only crds.yaml now.  changed the name to crdsYAML.